README.md

```diff
@@ -73,6 +73,7 @@ NEW: Be sure to check out the [Twitch directory listings](#twitch) at the end
 - [CodeRushed](#coderushed-mark-miller) - **streaming:** C#, .Net Core, ASP.Net Core, TypeScript, JavaScript, CodeRush dog-fooding
 - [Collin Henderson](#collin-henderson) - **streaming:** JavaScript, PHP, Laravel, Vue.js
 - [Cory Knox](#cory-knox) - **streaming:** PowerShell, C#, TypeScript, VSCode extension development
+- [Coding Garden with CJ](#Coding Garden with CJ) - **streaming** JavaScript, Vue.js, Node.js, React, Codewars problem solving.
```

Comment:
The anchor tag here is incorrect so it won't link to the proper point on the page. Instead, it needs to be `#coding-garden-with-cj`. Also, these are alphabetically sorted, so this entry (and the detailed block below) would go after the CodeRushed entry.

Reply:
My bad sorry. :)
Should be fixed.